lib/prototype-rails/on_load_action_view.rb

```diff
@@ -14,9 +14,11 @@
   include ActionView::Helpers::ScriptaculousHelper
 end
 
-ActionView::TestCase.class_eval do
-  include ActionView::Helpers::PrototypeHelper
-  include ActionView::Helpers::ScriptaculousHelper
+if defined?(Rails) && Rails.env.test?
```

Comment:
I mean `if defined?(Rails.env) && Rails.env.test?`.

I really thing we should not check any environment but I can't think in a better option besides a configuration.